docs/DeveloperGuide.md

```diff
@@ -312,16 +320,16 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Non-Functional Requirements
```

Comment:
Just a suggestion - what about adding an NFR for working at different screen resolutions and window sizes?

Reply:
Nice suggestion!